data/gaTable.yaml

```diff
@@ -515,7 +515,7 @@
   webPropertyId: UA-25350863-1
   internalWebPropertyId: '49291662'
   viewName: ''
-  timezone: ''
+  timezone: 'America/Denver'
```

Comment:
@wdwatkins could you go here:
https://github.com/USGS-VIZLAB/internal-analytics/blob/master/scripts/visualize/timeDayUse.R#L67

and get Denver (and any other non-central times that need correction) in there?